### PR TITLE
test: added unit tests for the DoNotDisturbUtils utility class

### PR DESCRIPTION
**Pull request by ardevd**

### CODE REVIEW

File: app/src/test/java/com/nextcloud/talk/utils/DoNotDisturbUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    @Test
+    public void shouldVibrate_givenVibratorAndRingerModeSilent_assertReturnsFalse() {
+        when(vibrator.hasVibrator()).thenReturn(true);
+
+        when(audioManager.getRingerMode()).thenReturn(AudioManager.RINGER_MODE_SILENT);
+
+        assertFalse("shouldVibrate returned true despite ringer mode set to silent",
+                DoNotDisturbUtils.shouldVibrate(true));
+    }
+}
```
Comment (mario) [inline app/src/test/java/com/nextcloud/talk/utils/DoNotDisturbUtilsTest.java]: New line :P

Comment (ardevd) [reply app/src/test/java/com/nextcloud/talk/utils/DoNotDisturbUtilsTest.java]: Fixed ;)

**Comment by ardevd** [comment]
bump? I believe this can be merged.

**Comment by mario** [comment]
@ardevd next week - hope you'll have fun holidays! :)

**Comment by ardevd** [comment]
> @ardevd next week - hope you'll have fun holidays! :)

No prob! Just wanted to make sure I wasn't missing anything. Have a great holiday!

**Comment by jancborchardt** [comment]
@ardevd we have a "Talk development chat" for all contributors by the way. I can add you with a guest account via your email address if you like – it would be nice to have you! :)

**Comment by ardevd** [comment]
Sure, sounds great!

**Pull request merged**